src/interface/src/app/login/login.component.html

```diff
@@ -14,16 +14,25 @@ <h1 class="signin-title">Sign in to Planscape</h1>
           <input type="password" required formControlName="password" matInput />
         </mat-form-field>
 
+        <mat-card-actions>
+          <a
```

Comment:
Why not a button? I seem to recall that a11y validators will reject <a> tags with (click) actions?

Reply:
The mocks that I was given showed that it was an anchor element, so I am following along with the mocks.